MatrixSDKTests/MXSpaceServiceTest.swift

```diff
@@ -169,4 +202,179 @@ class MXSpaceServiceTest: XCTestCase {
             }
         }
     }
+    
+    /// - Create Bob
```

Comment:
Can you report these comments in the code?
This is absolutely boring but it is even more boring to understand what a test does when you need to debug it afterwards.